libs/vkd3d/command.c

```diff
@@ -7142,7 +7151,16 @@ static HRESULT d3d12_command_list_init(struct d3d12_command_list *list, struct d
     HRESULT hr;
 
     memset(list, 0, sizeof(*list));
+
+#ifdef VKD3D_ENABLE_PROFILING
+    if (vkd3d_uses_profiling())
+        list->ID3D12GraphicsCommandList_iface.lpVtbl = &d3d12_command_list_vtbl_profiled;
+    else
+        list->ID3D12GraphicsCommandList_iface.lpVtbl = &d3d12_command_list_vtbl;
```

Comment:
Maybe just make vkd3d_use_profiling be some inline thing that returns 0 when not defined?

Reply:
Possible, but that means having to build in the profiling vtables and entry points always, which seems a bit useless.

Reply:
If only we had constexpr :(